mmf/utils/general.py

```diff
@@ -333,33 +332,6 @@ def get_sizes_list(dim, chunks):
     return sizes_list
 
 
-def get_max_updates(config_max_updates, config_max_epochs, train_loader, update_freq):
```

Comment:
As discussed before can we import these here as well and add `# noqa` to it so that flake8 doesn't complain.

Reply:
ok! Will move them here!